.github/workflows/ci.yml

```diff
@@ -25,17 +27,18 @@ jobs:
           ctest
           sudo make install
           # Install redis for tests against external Redis server
-          cd ~
-          git clone https://github.com/redis/redis.git
-          cd redis 
-          make -j 4
-
+                              
+          # Build Redis to run it against librdb tests
```

Comment:
@moticless You may want to consider getting a Redis binary, like we do [here](https://github.com/RedisLabs/memtier_benchmark/blob/4203084b085e0f93bb9130c281362ea651d2140c/.github/workflows/ci.yml#L29). It's potentially faster.

Reply:
Good reference. thanks.

Currently librdb is relying on Redis repo for building redis binary and reusing `testrdb.c` source to test modules parsing.
I know it is preferable to librdb have its own source to build testable module but i prefer to wait with that effort since I plan anyway (second phase?!) to expand callback module API to parse primitives as well (and not only RESTORE module).
All in all, additional  +1minute of CI time.